bin/pt-online-schema-change

```diff
@@ -8706,7 +8706,7 @@ sub main {
 
       # Issue a strong warning if alter-foreign-keys-method = none
       if ( $alter_fk_method eq 'none' && !$o->get('force') ) {
-         print STDERR "WARNING! Using alter-foreign-keys-method = \"none\". This will typically cause foreign key violations!\nThis method of handling foreign key constraints is only provided so that the database administrator can disable the tool’s built-in functionality if desired.\n\nContinue anyway? (y/N)";
+         print STDERR "WARNING! Using alter-foreign-keys-method = \"none\". This will typically cause foreign key violations!\nThis method of handling foreign key constraints is only provided so that the database administrator can disable the tool's built-in functionality if desired.\n\nContinue anyway? (y/N)";
```

Comment:
I suggest that this be rewritten to remove the apostrophe. Change to:
database adminstrator can disable the built-in functionality if
Inanimate objects (tool) cannot possess anything.

Reply:
```suggestion
         print STDERR "WARNING! Using alter-foreign-keys-method = \"none\". This will typically cause foreign key violations!\nThis method of handling foreign key constraints is only provided so that the database administrator can disable the built-in functionality if the tool cannot possess anything.\n\nContinue anyway? (y/N)";
```
I would prefer to avoid using not-so-frequently-used words as "Inanimate objects", so it is easier to understand for non-native English speakers.